google-cloud-graalvm-samples/logging-sample/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Logging Client Libraries with GraalVM
+
+This application uses the Google Cloud [Logging Client Libraries](https://github.com/googleapis/java-logging) and can be compiled with GraalVM Native Image.
+
+## Setup Instructions
+
+1. Follow the [GCP Project and GraalVM Setup Instructions](../README.md).
+
+2. Enable the [Logging APIs](https://console.cloud.google.com/flows/enableapi?apiid=logging.googleapis.com).
+
+## Run with GraalVM Compilation
+
+Navigate to this directory in a new terminal.
+
+1. Compile the application using the GraalVM Compiler. This step may take a few minutes.
+
+   ```
+   mvn package -P graal
+   ```
+
+2. Run the application:
+
+   ```
+   ./target/com.example.loggingsampleapplication
+   ```
+
+3. The application will log a message. Navigate to the [Cloud Console Logs Viewer](https://console.cloud.google.com/logs/viewer) to view the newly generated log entry.
```

Comment:
>will log a message.

To the console? To the Cloud console? Both? I'd specify the expectation.

I'd also specify what what was logged so someone who is using this with another noisy product could pick it out from their other noise.